build.gradle

```
@@ -6,7 +6,8 @@ buildscript {
     ext.coroutines_version = '1.5.2'
     ext.accompanist_version = '0.23.0'
     ext.room_version = "2.4.2"
-    ext.IS_DEVELOPMENT = System.getenv("CI") == null
+    // set IS_DEVELOPMENT to true if you want to use with local version of sdk
+    ext.IS_DEVELOPMENT = false
```

I am OK with opting in for local development when not on a CI. But the CI should always have IS_DEVELOPMENT false. 

```suggestion
    // set IS_DEVELOPMENT boolean value to true if you want to use with local version of sdk
    ext.IS_DEVELOPMENT = (System.getenv("CI") == null) && false 
```

---

this please, the above check takes a lot of my brain cells 😆  

```suggestion
    // set IS_DEVELOPMENT to true if you want to use with local version of sdk
   ext.IS_DEVELOPMENT = System.getenv("CI") ? false : true
```

---

currently the local behaviour was to use the local sdk i set the value to `false` in the spirit of making it easy to run without needing the sdk and this should work for the CI as well.

developers would need to explicitly set it to `true` to use with the local version of the sdk which also involves running the local maven server from the sdk project, hence why changing the default from `true to false`.

---

🤦 yup you are correct, it would work for CI too, as it would always be always false and can be set true when needed.